content/doc/administration/requirements/java.adoc

```diff
@@ -58,3 +58,18 @@ Modern versions of Jenkins controllers and Jenkins agents verify Java requiremen
 and notify users when they are launched with an unsupported version.
 
 The plugin:versioncolumn[Versions Node Monitors plugin] provides detailed Java version monitoring.
+
+== Java Development Kits Used
+
+Jenkins uses Eclipse Temurin as its primary JDK for building and testing Java based applications.
```

Comment:
Optional - hyperlink to the Eclipse Temurin project in case readers want to know more about it.
```suggestion
Jenkins uses link:https://projects.eclipse.org/projects/adoptium.temurin[Eclipse Temurin] as its primary JDK for building and testing Java based applications.
```